service-base/src/main/java/org/eclipse/hono/service/AbstractProtocolAdapterBase.java

```diff
@@ -387,24 +389,29 @@ protected final void closeClients(final Handler<AsyncResult<Void>> closeHandler)
     }
 
     /**
-     * Gets a registration status assertion for a device.
+     * Gets an assertion for a device's registration status.
+     * <p>
+     * The returned JSON object contains the assertion for the device
+     * under property {@link RegistrationConstants#FIELD_ASSERTION}.
+     * <p>
+     * In addition to the assertion the returned object may include <em>default</em>
+     * values for properties to set on messages published by the device.
```

Comment:
... under property {@link RegistrationConstants.FIELD_DEFAULTS}. ???

Reply:
In JavaDoc tags, the `#` is the delimiter between class and field/method name.